AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift

```diff
@@ -210,9 +214,9 @@ class CompleteScreenViewModel: ObservableObject {
             
             self.sessionStreams = .ready(self.externalSessionWithStreams!.streams.map {
                 .init(id: $0.id,
-                      sensorName: Self.getSensorName($0.sensorName),
+                      sensorName: self.getShortSensorName($0.sensorName),
                       sensorUnit: $0.unitSymbol,
-                      lastMeasurementValue: $0.measurements.last?.value,
+                      lastMeasurementValue: $0.measurements.last?.value != nil ? self.showConvertedValue($0.measurements.last!.value, sensorName: $0.sensorName) : $0.measurements.last?.value,
```

Comment:
This is a bit weird for me, I mean: you are checking if the value is not nil and after that, if it is - you are using it.